File: preview/sync/sync_list_permission/update-default/update-default.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$sync_list_permission = $twilio->preview->sync->services("ISaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
                                              ->syncLists("ESaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
                                              ->syncListPermissions("identity")
                                              ->update(True, True, True);

print($sync_list_permission.accountSid);

File: preview/sync/sync_list_permission/update-default/update-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

sync_list_permission = @client.preview.sync
                              .services('ISaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
                              .sync_lists('ESaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
                              .sync_list_permissions('identity')
                              .update(read: true, write: true, manage: true)

puts sync_list_permission.account_sid